tests/unit_tests/test_nn/test_converters/test_tensorflow/test_Upsample.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+from dnnv.nn.converters.tensorflow import *
+from dnnv.nn.operations import *
+
+
```

Comment:
Could you also add tests where each input to Upsample can be an Input operation to ensure that the operation works with non-concrete input values? You can duplicate one of the test cases below and use input operations instead of concrete values to construct the op graph, and then pass the required values in when you call the op graph to get the outputs.